performance/reports/rp.py

```diff
@@ -42,14 +42,16 @@ def get_df_successes_by_rp(df_verifications_by_rp):
 
 
 def export_metrics_to_csv(df_export, report_output_path, date_start):
-    if not os.path.exists(report_output_path):
-        os.makedirs(report_output_path)
+    report_output_path_for_week = os.path.join(report_output_path, date_start)
+    if not os.path.exists(report_output_path_for_week):
+        os.makedirs(report_output_path_for_week)
+
     # Create export file with all RPs data
-    df_export.to_csv(os.path.join(report_output_path, f'rp_report-{date_start}.csv'))
+    df_export.to_csv(os.path.join(report_output_path_for_week, f'{date_start}-_All-RPs-rp_report.csv'))
```

Comment:
I reckon the capitalisation is a bit ugly myself, and if the date is in the dirname then maybe we don't need the date in the filename too? So perhaps just `_all.csv` if we're also going with the above suggestion re dirnames.